wadsrc/static/zscript/actors/shared/camera.zs

```diff
@@ -212,3 +212,35 @@ class SpectatorCamera : Actor
 		}
 	}
 }
+
+Class OrthographicCamera : Actor
+{
+	Default
+	{
+		+NOBLOCKMAP
+		+NOINTERACTION
+		CameraHeight 0;
+		RenderStyle "None";
+	}
+
+	override void PostBeginPlay()
+	{
+		Super.PostBeginPlay();
+		UpdateViewPos();
+	}
+	
+	override void Tick()
+	{
+		if (current != args[0])
+			UpdateViewPos();
+		
+		Super.Tick();
+	}
+
+	private int current;
+	private void UpdateViewPos()
```

Comment:
Don't use a private function in a virtual method. Those should be set to protected

Reply:
I see no reason to, but I went ahead and did as asked.

Reply:
If they're private they cannot be modified from child class overrides. If the behavior shouldn't be modifiable it's better to bundle it in a protected function and call that from Tick() instead